app/views/layouts/angular-bootstrap/_auth_credentials_angular_bootstrap.html.haml

```diff
@@ -18,14 +18,14 @@
 - main_scope            = vm_scope ? "$parent.vm" : "$parent"
 
 %div{"ng-controller" => "CredentialsController as vm", "vm-scope" => "$parent.vm.model ? $parent.vm : $parent"}
-  %div{"ng-show" => "#{ng_show} && #{ng_show_userid}"}
+  %div{"ng-show" => "#{ng_show} && #{ng_show_userid} && emsCommonModel.emstype != 'kubevirt'"}
```

Comment:
This feels like the wrong approach to preventing things to appear.  `!= <provider>` feels like a red flag.  Is this saying there won't be credentials?  If so, that doesn't make any sense to me.

Reply:
No provider specific entries in this haml, as mentioned in https://github.com/ManageIQ/manageiq-ui-classic/pull/3051#issuecomment-351889445